spec/models/post_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  before(:all) do
+    @user = User.create(
+      name: 'Ambrose',
+      photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+      bio: 'I am a preacher of CHRIST',
+      posts_counter: 0
+    )
+    @post = Post.create(
+      title: 'Programming school at Microverse is awesome',
+      text: 'Coding is fun',
+      comments_counter: 0,
+      likes_counter: 1,
+      author_id: @user.id
+    )
+  end
+
+  it '@post created should be valid' do
+    expect(@post).to be_valid
+  end
+  it '@post created should be valid' do
```

Comment:
- [ ] Great job. kindly rename some of the custom tests, so methods in the models shoudl have a corresponsding unit test
eg instead of putting all `it '@post created should be valid' do`  also give the other titles like
* Title must not be blank.
* Title must not exceed 250 characters.
* CommentsCounter must be an integer greater than or equal zero.
* LikesCounter must be an integer greater than or equal zero.